api/templates/design.go

```diff
@@ -920,6 +924,14 @@ var Input = Type("input", func() {
 		Example("input")
 	})
 	Attribute("default", Any, "Default value, match defined type.", func() {
+		Meta(oneof.Meta, json.MustEncodeString([]*openapi.Schema{
+			{Type: openapi.String},
+			{Type: openapi.Integer},
+			{Type: "double"},
+			{Type: openapi.Boolean},
+			{Type: openapi.Array, Items: &openapi.Schema{Type: openapi.String}},
+			{Type: openapi.Object},
+		}, false))
 		Example("foo bar")
 	})
```

Comment:
- Goa umoznuje generovat HTTP REST a gRPC APIs.
- Goa podporuje OneOf, ale zial nepodporuje array typ, ktory potrebujeme, vid: https://keboola.atlassian.net/browse/KAC-124?focusedCommentId=44641
- Je to preto, lebo sa to neda implementovat pre gRPC, ktore sice NEpouzivame, ale v Goa sa tato chyba neda ignorovat.
- Teda, po funkcnej stranke je vsetko OK, ale UI potrebuje mat v OpenAPI.json konkretne typy.
- Preto si tu nad dany atributom definujeme `Meta` = `oneof.Meta = "openapi:extension:x-one-of"`.
- To nam prida to OpenApi definiciu `x-one-of`. .... no my potrebujeme `oneOf`.
- Go umoznuje pridavat vlastne fields/`OpenApi extensions` iba s prefixom `x-`.
- Preto som vytvoril jednoduchu `Goa extension`, ktora premenuje `x-one-of` -> `oneOf`, vid dalej.
- Je to najkratsi sposob ako dosiahnut zelany vysledok.